.github/workflows/vast.yaml

```diff
@@ -775,11 +775,11 @@ jobs:
       matrix:
         vast:
           - os: ubuntu-20.04
-            container: python:3.10-slim-bullseye
+            container: debian:bookworm-slim
             name: Debian
             compiler: GCC
-            cc: gcc-10
-            cxx: g++-10
+            cc: gcc-12
```

Comment:
I think we should update the table in the docs in build.md already to gcc12+ and clang17+, as those are now the only versions we actively test with.

Reply:
Nice, i am actually struggling with my code not working on gcc 10 due to a bug. I am trying to update to gcc 11 myself. Can't wait for it to get merged :D